README.md

```diff
@@ -2,6 +2,12 @@
 
 We will be using this repo to create an API for our entire class. We will use Flask for this application and each member of the class will work within this single repo.
 
+<<<<<<< HEAD
+# How to use the Quotes Endpoint
+
+Open up the url to http://127.0.0.1:5000/quotes
+Start reading people's favorite quotes!
+=======
```

Comment:
The merge text `<<<<<<<< HEAD` and `>>>>>>>> main` should be removed before this code is merged.